notifier/selfstate/heartbeat/notifier.go

```diff
@@ -3,29 +3,42 @@ package heartbeat
 import (
 	"fmt"
 
+	"github.com/moira-alert/moira/metrics"
+
 	"github.com/moira-alert/moira"
 )
 
 type notifier struct {
-	db  moira.Database
-	log moira.Logger
+	db              moira.Database
+	log             moira.Logger
+	notifierIsAlive metrics.Meter
 }
 
-func GetNotifier(logger moira.Logger, database moira.Database) Heartbeater {
+func GetNotifier(logger moira.Logger, database moira.Database, notifierIsAlive metrics.Meter) Heartbeater {
```

Comment:
Maybe we sholud make a separate struct for all heartbeater metrics in case we would like to to add some more later?

Reply:
i think later, now selfcheck only in notifier and i'm not sure, that it's ok monitor "filter" in "notifier", than "notifier-pod" is off will be no metric about filter, soooo it's not ok